modules/gdscript/gdscript_function.cpp

```diff
@@ -1479,16 +1479,21 @@ Variant GDScriptFunction::call(GDScriptInstance *p_instance, const Variant **p_a
 
 #ifdef DEBUG_ENABLED
 				GET_VARIANT_PTR(test, 1);
+				GET_VARIANT_PTR(message, 2);
 				bool result = test->booleanize();
 
 				if (!result) {
-
-					err_text = "Assertion failed.";
+					const String message_str = *message;
```

Comment:
curious question: how is String in godot implemented? wouldn't it be better to use `const String&` here, as to not create copies? Or is it safe and efficient to copy, because of CoW?

Reply:
`const String &` should be fine yes.